runpod/serverless/modules/rp_http.py

```diff
@@ -46,6 +48,9 @@ async def send_result(session, job_data, job):
         await transmit(session, job_data, job_done_url)
         log.debug(f"{job['id']} | Results sent.")
 
+        log.info(f'{job["id"]} | Finished')
```

Comment:
Nitpicking, but I try to place the log after the final action. In this case, "finished" should indicate the job is done and has been removed from the list. 

If, for whatever reason, the job is not removed then we have a misleading log.

Reply:
I like this philosophy. I'll update `send_result` log "finished" only after the job has been removed from the list (and of course, sent to the server successfully).